exercises/exercise-1/exercise-1.js

```diff
@@ -4,10 +4,16 @@ const personOne = {
   favouriteFood: "Spinach",
 };
 
-function introduceYourself(___________________________) {
+const {name, age, favouriteFood} = personOne;
+
+
+function introduceYourself(object) {
```

Comment:
So what you are doing here is destructuring the object into global variables, which works. But a better idea would be to destruct the object inside the functions parameters so they can only be used in the function like so: 

```suggestion

function introduceYourself({name, age, favouriteFood}) {
```